src/app/queries.js

```diff
@@ -19,21 +18,10 @@ export const DELETE_CARD = gql`
 `;
 
 export const NEW_CARD = gql`
-  mutation newCard(
-    $name: String
-    $listId: ID
-    $description: String
-    $position: Int
-  ) {
-    newCard(
-      name: $name
-      listId: $listId
-      description: $description
-      position: $position
-    ) {
+  mutation newCard($body: String!, $listId: ID, $position: Int) {
```

Comment:
The SQL has `body` as optional, looks like here it's required?

Reply:
yeah true, we should probably make it `not null` in the SQL too as I don't think we'd ever want a card with no body

Reply:
Agreed